Protos/V1/filters_common.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package armonik.api.grpc.v1;
+
+import "google/protobuf/timestamp.proto";
+import "result_status.proto";
+import "session_status.proto";
+import "task_status.proto";
+
+option csharp_namespace = "Armonik.Api.gRPC.V1";
+
+enum FilterStringOperator {
+  FILTER_STRING_OPERATOR_UNSPECIFIED = 0; /** Unspecified */
+  FILTER_STRING_OPERATOR_EQUAL = 1; /** Equal */
+  FILTER_STRING_OPERATOR_NOT_EQUAL = 2; /** Not equal */
+  FILTER_STRING_OPERATOR_CONTAINS = 3; /** Contains */
+  FILTER_STRING_OPERATOR_NOT_CONTAINS = 4; /** Not contains */
+  FILTER_STRING_OPERATOR_STARTS_WITH = 5; /** Starts with */
+  FILTER_STRING_OPERATOR_ENDS_WITH = 6; /** Ends with */
```

Comment:
I would remove `UNSPECIFIED`, and make sure `EQUAL` is with index 0 on all filters.

Reply:
agree, will update this.